graphql/api.js

```diff
@@ -77,7 +77,7 @@ export const createAccount = async (email, name, company, role, date, type, note
               company: $company  
               role: $role
               initialExcitement: 1
-              currentExcitment: 3 
```

Comment:
could we update this to pull in the actual excitement data instead of this hard coded number?

Reply:
we could, it's sort of unrelated to this but should only take a few lines

Reply:
and yeah i manually updated the records considering that there's only a few